packages/ui/src/components/Form/customField/CustomLongTextField.tsx

```diff
@@ -1,9 +1,11 @@
-import { HTMLFieldProps } from 'uniforms';
 import { LongTextField } from '@kaoto-next/uniforms-patternfly';
+import { FunctionComponent, RefObject } from 'react';
 import './CustomLongTextField.scss';
 
-export type CustomLongTextFieldProps = HTMLFieldProps<string, HTMLDivElement>;
+type LongTextFieldProps = Parameters<typeof LongTextField>[0] & { ref: RefObject<HTMLTextAreaElement> };
 
-export const CustomLongTextField = (props: CustomLongTextFieldProps) => {
-  return <LongTextField className="custom-long-test-field" {...props} rows={1} autoResize={true} />;
+export const CustomLongTextField: FunctionComponent<LongTextFieldProps> = (props) => {
+  return (
+    <LongTextField className="custom-long-test-field" {...props} inputRef={props.ref} rows={1} autoResize={true} />
```

Comment:
How does the `CustomLongField` Loads perfectly fine for some fields(like description) without a `ref`?